python/cross_service/resilient_service/runner.py

```diff
@@ -93,6 +94,7 @@ def deploy(self):
             print("Couldn't connect to the load balancer, verifying that the port is open...")
             current_ip_address = requests.get('http://checkip.amazonaws.com').text.strip()
             sec_group, port_is_open = self.autoscaler.verify_inbound_port(vpc, self.port, current_ip_address)
+            sec_group, ssh_port_is_open = self.autoscaler.verify_inbound_port(vpc, self.ssh_port, current_ip_address)
```

Comment:
Add a call to `self.param_helper.reset()` at line 89 (just before verifying the load balancer) in order to set up parameters.

Reply:
Or possibly even a little sooner--before creating the load balancer--so that the parameters are established before there's any possibility of something pinging the web server.

Reply:
Added.